docs/testing/save-for-later.md

```diff
@@ -32,4 +32,18 @@
 2) Under the `Authorization` tab, on the type dropdown select `Bearer Token` and paste in the token generated from the Okta auth
    ![Authorization](../images/Authorization.png)
 3) Under the `Headers` tab add `x-gu-is-oauth: true`
-   ![Headers](../images/GetHeaders.png)
\ No newline at end of file
+   ![Headers](../images/GetHeaders.png)
+
+
+## Testing locally
+
+### Pre-requisites
+1) Mobile credentials from [Janus](https://janus.gutools.co.uk/login)
+2) Follow the authentication flow as outlined at the [top](#setting-up-oauth-flow)
+
+* Go to `src/main/scala/com/gu/sfl/local/LocalServer.scala`
+* Select the drop down menu to the left of the run button and select `LocalServer` under run configurations
+* Then choose `Edit Configurations`
```

Comment:
In my IntelliJ I had to do Edit Configurations -> select `com.gu.sfl.local.LocalServer` in the module, which then populated LocalServer